tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -688,34 +688,15 @@ void StorageManagerCanonical::delete_array(const char* array_name) {
       std::numeric_limits<uint64_t>::max());
 
   // Get the metadata and schema uris to be deleted
-  /* Note: metadata files may not be present, try to delete anyway */
-  const auto& array_meta_uris = array_dir.array_meta_uris();
-  const auto& fragment_meta_uris = array_dir.fragment_meta_uris();
-  const auto& array_schema_uris = array_dir.array_schema_uris();
+  // Note: metadata files may not be present, try to delete anyway
```

Comment:
We should change to auto&, or remove the variable declarations and call the functions directly with the arraydir calls, ie:
`vfs_->remove_files(compute_tp_, array_dir.array_meta_uris());`

Reply:
Do the same change everywhere remove_files is called in this PR.